docs/cloud-native-security/kspm-faq.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[kspm-faq]]
+== KSPM FAQ
+Frequently asked questions related to the Kubernetes Security Posture Management (KSPM) integration.
+
+[discrete]
+== General KSPM FAQ
+
+*Can I evaluate the security posture of my Amazon EKS clusters?*
+Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters.
+
+*How often is my cluster’s security posture evaluated?*
+Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.
```

Comment:
What do you think about combining the "General KSPM FAQ" section with the Cluster enrollment FAQ" section? Both sections seem to address common questions about clusters, so maybe you could group them together under a common section titled "Cluster enrollment and security", "Cluster monitoring", or just "Clusters".